cost/turbonomics/allocate_virtual_machines_recommendations/aws/README.md

```diff
@@ -0,0 +1,28 @@
+# Turbonomic Allocate Virtual Machine Recommendations AWS
+
+## What it does
+
+The Turbonomic Allocate Virtual Machines Recommendations AWS policy utilizes Turbonomic [Credentials](https://turbonomic.com/api/v3/markets/{market_uuid}/actions) endpoint to provide AWS VM allocation recommendations for Reserved Instance coverage. From these recommendations we provide monthly savings estimates based on Turbonomic per hour costs
```

Comment:
```suggestion
The Turbonomic Allocate Virtual Machines Recommendations AWS policy utilizes Turbonomic Credentials (https://turbonomic.com/api/v3/markets/{market_uuid}/actions) endpoint to provide AWS VM allocation recommendations for Reserved Instance coverage. From these recommendations we provide monthly savings estimates based on Turbonomic per hour costs
```

There is no reason to hyperlink to a URL if the URL is not valid.  This is why some of the Danger errors are being raised.

This seems like a good place to mention the name of the Turbonomic API that is being used and link to the docs for the Turbonomic API (which I don't see anywhere?)

Same for all other occurrences where you have `[Credentials](https://xxxx.turbonomic.com/api/.....)`

Reply:
@bryankaraffa - Done my friend!